examples/speaker_tasks/diarization/conf/inference/diar_infer_general.yaml

```diff
@@ -52,7 +54,9 @@ diarizer:
       max_rp_threshold: 0.25 # Determines the range of p-value search: 0 < p <= max_rp_threshold. 
       sparse_search_volume: 10 # The higher the number, the more values will be examined with more time. 
       maj_vote_spk_count: False  # If True, take a majority vote on multiple p-values to estimate the number of speakers.
-  
+      sub_cluster_n: 50 # Number of sub-clusters for each sub-window for long-form audio clustering.
+      unit_window_len: 10000 # Window length for the unit of chunking. Control according to the GPU-memory capacity. 
```

Comment:
Can we change this to something like `chunk` or `sub_utterance`?

Reply:
Changed to `chunk_cluster_count` and `embeddings_per_chunk`.